napari/layers/_layer_actions.py

```diff
@@ -67,15 +67,26 @@ def _project(ll: LayerList, axis: int = 0, mode='max'):
     # this is not the desired behavior for coordinate-based layers
     # but the action is currently only enabled for 'image_active and ndim > 2'
     # before opening up to other layer types, this line should be updated.
-    data = (getattr(np, mode)(layer.data, axis=axis, keepdims=True),)
+    data = (getattr(np, mode)(layer.data, axis=axis, keepdims=False),)
     layer = cast('Image', layer)
-    meta = {
-        **layer._get_base_state(),
+    # get the meta data of the layer, but without transforms
+    layer_meta = {
+        key: layer._get_base_state()[key]
+        for key in layer._get_base_state()
+        if key not in ('scale', 'translate', 'rotate', 'shear', 'affine')
+    }
+    # make metadata for the projection without transforms--will add them back later
+    proj_meta = {
+        **layer_meta,
         'name': f'{layer} {mode}-proj',
         'colormap': layer.colormap.name,
         'rendering': layer.rendering,
     }
```

Comment:
I would find it less confusing like this:

```suggestion
    meta = {
        key: layer._get_base_state()[key]
        for key in layer._get_base_state()
        if key not in ('scale', 'translate', 'rotate', 'shear', 'affine')
    }
    meta.update({
        'name': f'{layer} {mode}-proj',
        'colormap': layer.colormap.name,
        'rendering': layer.rendering,
    })
```

but it't just an aesthetic choice, so feel free to ignore.

Reply:
I was trying to match what was there. Didn't know about `.update()` seems like the better way to do this, especially since the original layer metadata isn't useful to keep in a var.